template-parts/paragraph.php

```diff
@@ -0,0 +1,7 @@
+<?php if( get_field('enable_paragraph') && get_field('paragraph') ): ?>
+  <div class='row'>
+    <div class='col-12 col-md-8 m-auto'>
```

Comment:
Sorry I'm going to be picky here, let's use double quotes when we assign values to attributes.